benchmarks/ai-benchmark/Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM tensorflow/tensorflow:2.4.1-gpu
+# This Dockerfile is used to build a Docker image for running the AI Benchmark.
+# It is based on the tensorflow/tensorflow:latest-gpu image.
 
-RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
-
-RUN pip install --upgrade pip
-
-RUN apt-get -y install git
-RUN git clone -b feat/transformer https://github.com/shiyoubun/ai-benchmark.git
+FROM tensorflow/tensorflow:latest-gpu
 
+# Set the working directory to /ai-benchmark
 WORKDIR ai-benchmark
-RUN pip install -e .
 
-ENTRYPOINT [ "python", "bin/ai-benchmark.py" ]
+# Update the package list and install git and apt-utils
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends apt-utils git && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install --no-cache-dir --upgrade pip && \
+    git clone https://github.com/haitwang-cloud/ai-benchmark . && \
```

Comment:
Can you explain why you use your own repo? Don't know the context here

Reply:
Hi there, it seems that the previous implementation isn't functioning as expected in my tests, which I believe is due to a lack of maintenance. As a result, I've taken the initiative to make some enhancements myself. I'd be more than happy to contribute these improvements back to the Project-HAMi.